src/bowling_SC.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+class bowlingGame {
+
+  constructor() {
+    this.rolls = [[], [], [], [], [], [], [], [], [], []];
+    this.runningScore = 0;
+    this.frameIndex = 1;
+    this.rollIndex = 0;
+  };
+
+  firstRoll(points) {
+    this.rolls[this.frameIndex -1].push(points);
+    this.runningScore += points;
+    if (points === 10) {
+      this.frameIndex++
+    };
+  };
+
+  secondRoll(points) {
+    this.rolls[this.frameIndex -1].push(points);
+    this.runningScore += points;
+    this.frameIndex++
+  };
+
+  spareCalc() {
+    let bonus = 0;
+    const frameTotal = this.rolls[this.frameIndex - 2][this.rollIndex] + this.rolls[this.frameIndex - 2][this.rollIndex + 1];
```

Comment:
Is this the same calculation done in `strikeBonus`? Might there be another name for `strikeBonus` that would allow us to use that function here?